Makefile

```
@@ -456,59 +500,59 @@ ci.e2e.build.all: ci.e2e.build.centos-8
 ci.e2e.build.all: ci.e2e.build.ubuntu-18
 ci.e2e.build.all: ci.e2e.build.ubuntu-20
 ci.e2e.build.all: ci.e2e.build.sles-15
-ci.e2e.build.all: ci.e2e.build.centos-7-nvidia
-ci.e2e.build.all: ci.e2e.build.centos-8-nvidia
-ci.e2e.build.all: ci.e2e.build.sles-15-nvidia
 ci.e2e.build.all: ci.e2e.build.oracle-7
 ci.e2e.build.all: ci.e2e.build.oracle-8
 ci.e2e.build.all: ci.e2e.build.flatcar
+ci.e2e.build.all: ci.e2e.build.centos-7-nvidia
+ci.e2e.build.all: ci.e2e.build.centos-8-nvidia
+ci.e2e.build.all: ci.e2e.build.sles-15-nvidia
 
 # Run an E2E test in its own devkit container.
 ci.e2e.build.%:
 	make devkit.run WHAT="make e2e.build.$*"
 
-e2e.build.centos-7: build
-	./bin/konvoy-image build images/ami/centos-7.yaml -v ${VERBOSITY}
+e2e.build.centos-7:
+	make centos7
 	make docker.clean-latest-ami
 
-e2e.build.centos-8: build
-	./bin/konvoy-image build images/ami/centos-8.yaml -v ${VERBOSITY}
+e2e.build.centos-8:
+	make centos8
 	make docker.clean-latest-ami
```

What about using dependencies instead of nested makes?

```suggestion
e2e.build.centos-8: centos8 docker.clean-latest-ami
```

---

Great idea!